resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -206,7 +185,9 @@ public DnsNameResolverBuilder resolvedAddressTypes(InternetProtocolFamily... res
      *
      * @param resolvedAddressTypes the address types
      * @return {@code this}
+     * @deprecated use {@link #resolvedAddressTypes(ResolvedAddressTypes)} instead
      */
+    @Deprecated
```

Comment:
this class is marked as unstable ... should we make the conversion routine `computeResolvedAddressTypes` available somewhere instead and call it out in the comments for the new method?